CONTRIBUTING.rst

```diff
@@ -674,36 +896,52 @@ Pull Requests
   rejection of the PR. The final consolidation eliminates both ``fixup!`` and
   ``drop!`` commits.
 
+Status checks
+-------------
+
 * We usually don't request a review before all status checks are green. In
   certain cases a preliminary review of a work in progress is permissible but
   the request for a preliminary review has to be qualified as such in a comment
   on the PR.
-  
+
+Holding branches warm
+---------------------
+
+* Some PR branches are can't be reviewed or merged for concerns external to the
```

Comment:
Consider removing the second sentence of this bullet. It reads fine to me without it.